client/routes/workflow/history.vue

```diff
@@ -258,17 +270,33 @@ export default {
         i.eventIds[i.eventIds.length - 1]
       );
     },
-    toggleShowGraph() {
-      if (this.showGraph) {
-        this.$router.replace({ query: omit(this.$route.query, 'showGraph') });
+    toggleShowDagGraph() {
+      if (this.graphView === 'dag') {
+        this.$router.replace({
+          query: omit(this.$route.query, 'graphView'),
+        });
+      } else {
+        this.$router.replace({
+          query: { ...this.$route.query, graphView: 'dag' },
+        });
+      }
+    },
+    toggleShowTimeline() {
+      if (this.graphView === 'timeline') {
```

Comment:
Having a graphView we are essentially limiting showing either timeline OR dag. This is probably fine as both would take too much screen space. But maybe it is better to give this choice to the user and leave flexibility to have both, at least from code perspective.

Reply:
I think this can be done however it would mean we would need two separate sliders to control which adds a lot more complexity. Perhaps we can add a task in the backlog and prioritize this against other work?

Reply:
Created https://t3.uberinternal.com/browse/CDNC-1057 to reflect work needed to complete this task.

Reply:
👍